### PR TITLE
Remove unused var

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: src/Downloader.php [hunk 1]
```diff
@@ -79,7 +79,6 @@ public function setRepositories(array $repos)
      */
     public function getRecipes(array $operations): array
     {
-        $max = 1000;
```
Comment (fabpot) [inline src/Downloader.php]: I would prefer to keep the var and replace the hardcoded `1000` in the code instead.

Comment (Pierstoval) [inline src/Downloader.php]: But this var is unused 🤔

Comment (chalasr) [inline src/Downloader.php]: > replace the hardcoded 1000 in the code instead

by `$max` so that it would be (used)

Comment (stof) [inline src/Downloader.php]: We could even use a private constant rather than a var

Comment (dunglas) [inline src/Downloader.php]: Switched to a private const

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request merged**